org.qgis.qgis-ltr.json

```
@@ -0,0 +1,614 @@
+{
+    "id" : "org.qgis.qgis-ltr",
+    "branch" : "stable",
+    "base": "io.qt.qtwebkit.BaseApp",
+    "base-version": "5.12",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "5.12",
```

Update runtime to 5.15
```suggestion
    "base-version": "5.15",
    "runtime" : "org.kde.Platform",
    "runtime-version" : "5.15",
```